android/build.gradle

```diff
@@ -2,13 +2,17 @@ apply plugin: 'com.android.library'
 
 import groovy.json.JsonSlurper
 
+def safeExtGet(prop, fallback) {
+    return rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
 android {
-    compileSdkVersion 28
+    compileSdkVersion rootProject.ext.get('compileSdkVersion')
```

Comment:
you need the fallbacks or this is going to error out.

Reply:
Thanks I am testing something with our integration tests failing.